### ISSUE TITLE
Log: rows are not colored

### ISSUE THREAD

**Issue by ildar170975** [home-assistant/frontend#22796]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

Rows for warnings & errors have same color.

Before 2024.11:
![3d1ade49a9f30a2ae4219adbcdc95e80a268f36f](https://github.com/user-attachments/assets/c24b2217-3028-4a19-b3d5-7955f862d4f5)

2024.11:
![552179ba69f27f9ee56e57290011efe37ce80aa1](https://github.com/user-attachments/assets/0e11a614-b577-4a1a-8426-e3e9cf86dbc9)

### Describe the behavior you expected

Rows for warnings & errors have different colors

### Steps to reproduce the issue

Open a Log

### What version of Home Assistant Core has the issue?

2024.11.2

### What was the last working version of Home Assistant Core?

2024.10.x

### In which browser are you experiencing the issue with?

Chrome 130.0.6723.117

### Which operating system are you using to run this browser?

Win10x64

**Comment by ildar170975** [comment]
Created a new test setup with 2025.1, right after onboarding came to Logs ans see this:

![Image](https://github.com/user-attachments/assets/c508cc93-514c-4b9a-a328-5c28f70b607a)

No colors.

**Comment by ildar170975** [comment]
And same bug in safe mode for that fresh empty setup (simulated an error in a template sensor):

![Image](https://github.com/user-attachments/assets/ce9b6d52-e3d5-4668-b213-017172297a5c)

**Comment by ildar170975** [comment]
Cannot say that I am surprised that this issues has no other comments than mine.
Colored logs are not shown in ANY of my docker setups.

**Comment by wendevlin** [comment]
I am also not able to reproduce it 😒

**Comment by wendevlin** [comment]
The logs have a color code and we use them. This hasn't changed since 5 years: https://github.com/home-assistant/frontend/blob/c73a9fccb825b4c6ea28bb908256777a74ad5661/src/components/ha-ansi-to-html.ts#L196

Can it be related to your installation method? 🤔

**Comment by wendevlin** [comment]
I can reproduce it on a non HAOS core. I'll have a look into it 👍

**Comment by wendevlin** [comment]
> Using a bot to close unsolved issues and ignoring proposals to change this situation is an intended way to bury existing problems.

As an active contributor you should know that we have a very big amount of issues and if we won't have this bot the number would be endless. The but is useful to remind and if the issues was solved in the meantime it will be closed automatically. 

I did looked into this issue and I can reproduce it. The issue is that core sends a different log format then supervisor. Supervisor adds special chars for coloring and core don't. So I think we need to add this extra information in the core endpoint.

**Comment by ildar170975** [comment]
@wendevlin 
Thanks a lot for your feedback. 
And once again thanks for your constant help in my PRs. 

As for the “stale” - marking an issues as “stale” and then closing it is not the problem. Totally agree that users tend to leave their issues open when they are resolved or obsolete. 
The problem is inability to reopen the closed issue. It was highlighted many times but it is still there.
To reopen an issue, a user either writing in the closed issue smth like “please reopen “ - and in my practice it was reopened only once. 
Or posting in Discord in dev channel (no idea if it still exists) something like “please reopen these issues” - and in my practice it helped not always (and thanks to Simon for his help there), sometimes my requests were not seen. Not to mention that Discord currently is not available in my fashist Russia and some other countries as well.
Another option is to create a new issue - but it may loose useful comments. 
In some core issue I was told that it is not possible to reopen a closed stale issue by a user due to GitHub limitations (I.e. ”it is not our fault, it is Guthub”). But advanced users (like you) still can reopen these issues - so I guess this can be altered to allow simple users (at least to an issue creator) to reopen.

**Comment by ildar170975** [comment]
2025.11.3 - still present
Please reopen

**Comment by wendevlin** [comment]
Yeah sorry it's a core issue, they would need to provide the color code prefixes per line as superviser is doing it. May you try to open the issue there, but I know there isn't a high priority on this :/

**Comment by ildar170975** [comment]
@wendevlin 
Thanks a lot for a feedback! I will try to investigate it when will have enough time, just wanted to have this problem not forgotten.
